src/pages/DetailsPage.js

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect, useState } from "react";
+
+import Badge from "../components/Badge";
+import Header from "../components/Header";
+import Icon from "./../components/Icon";
+
+const DetailsPage = (props) => {
+	const pokemonId = props.match.params.pokemonId;
+	const [pokemon, setPokemon] = useState({});
+	const getPokemonById = async (pokemonId) => {
+		let pokemon = {};
+		try {
+			pokemon = await fetch(
```

Comment:
Here you are mixing async/await with Promise...then. You should stick to async/await since it makes code easier to read :)